sandbox

```diff
@@ -215,7 +215,13 @@ sandbox () {
       VERBOSE_FLAG=" --verbose"
     fi
 
-    $DOCKER_COMPOSE$VERBOSE_FLAG -f "$SANDBOX_DIR/docker-compose.yml" "$@"
+    $DOCKER_COMPOSE$VERBOSE_FLAG -f "$SANDBOX_DIR/docker-compose.yml" "$@" algod
```

Comment:
BTW... if we do go with this approach, we ought to deprecate `$INDEXER_DISABLED`. Not sure the best way to go forward with such a deprecation. Sandbox is quite widely used so care needs to be taken.

Reply:
hmm maybe this isnt the best approach then, maybe docker config files per service?

@winder does this get easier with the docker stuff you're working on?

Reply:
My last suggestion doesn't really work because we have service dependency. I.e., even if you run:

```sh
docker compose up indexer indexer-db
```
you'll still get an `algod` running because of [this docker-compose.yml service dependancy](https://github.com/algorand/sandbox/blob/master/docker-compose.yml#L50).

So I think we're better of sticking with `INDEXER_DISABLED` and it's more of a communication issue about how to broadcast the change to the 0.1% who are currently using it yet relying on the `indexder-db`.